database/sqlite.go

```diff
@@ -129,7 +119,15 @@ func CreateTestSqliteDB(t *testing.T) *TestSQLiteDB {
 
 	ctx, cancelFunc := context.WithCancel(context.Background())
 
-	db, err := sqliteConnection(log.NewNopLogger(), filepath.Join(dir, "paygate.db")).Connect(ctx)
+	// logger = log.NewNopLogger()
+	logger := log.NewDefaultLogger()
+
+	db, err := sqliteConnection(logger, filepath.Join(dir, "paygate.db")).Connect(ctx)
```

Comment:
This is a copy and paste and it should probably be cleaned up. Notice the `paygate.db` sitting there. When I originally set this up it was copy and pasted from paygate and morphed to fit into `identity` and then coped into here. I think now that its in its final resting place we should clean this up. Also these `Test{MySQL|SQLite}DB structs aren't used in the templater code.

Reply:
During transition customers to "templater" I'll remove it. 

Btw, how does templater launche MySQL instance to test agains mysql db?

Reply:
It loads it up via docker-compose. When you run `make test` or `make check` it runs `docker-compose up`

Reply:
A single running external instance works great for this since we're building multi-tenancy it should maintain that one test using say tenant A shouldn't interfere with another test using say tenant B.

Reply:
do we have an example of such mysql / testing / configuration?

right now in customers and paygate we use dockertest and it launches container.